app/buyers/views/buyers.py

```diff
@@ -362,6 +363,11 @@ def publish_brief(framework_slug, lot_slug, brief_id):
             question_and_answers['slug'] = section['id']
 
     unanswered_required, unanswered_optional = count_unanswered_questions(sections)
+
+    if sections.get_section('set-how-long-your-requirements-will-be-live-for') and \
+            sections.get_section('set-how-long-your-requirements-will-be-live-for').questions[0].answer_required:
+            unanswered_required -= 1
+
     if request.method == 'POST':
         if unanswered_required > 0:
```

Comment:
So we don't want to abort if they have answered all questions apart from choosing how long their requirements will be live for?